app/src/sharedTest/java/org/oppia/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -79,9 +79,14 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class TopicLessonsFragmentTest {
 
-  @Inject lateinit var context: Context
-  @Inject lateinit var profileTestHelper: ProfileTestHelper
-  @Inject lateinit var storyProgressTestHelper: StoryProgressTestHelper
+  @Inject
+  lateinit var context: Context
```

Comment:
Bring this to single line.

Reply:
Ktlint is a bit different, it says to be on the next line. Also, Ben discuss the same in yesterday's meeting